runtime/remote_config.go

```diff
@@ -131,11 +132,15 @@ func (rc *remoteConfig) loadConfig() RemoteConfigMap {
 func (rc *remoteConfig) getValidatedValue(key string, vt jsonparser.ValueType) (*RemoteConfigValue, bool) {
 	ret, ok := rc.loadConfig()[key]
 	if !ok {
-		rc.logger.Warn(fmt.Sprintf("Key <%s> missing", key))
+		rc.logger.With(zap.String("key", key)).Warn("Key is missing")
```

Comment:
no need to call `With()` i think that will allocate a child logger.

Can just do `rc.Logger.Warn("Key is missing", zap.String("key", key))`

Reply:
sounds good, does this apply to all logging cases?